etna/transforms/math/lags.py

```diff
@@ -126,10 +138,10 @@ def __init__(self, lag: Union[int, Literal["auto"]], horizon: Optional[int] = No
         self.horizon: Optional[int] = None
         self._auto = False
 
-        self._freq: Optional[str] = None
+        self._freq: Optional[str] = _DEFAULT_FREQ  # type: ignore
```

Comment:
Why can't we leave `None` as a default value?

Reply:
Because now `None` is a valid value for `freq` which means that we use integer timestamp.

Reply:
Decided to leave as is to maintain consistency with other places